src/modules/google-organization-kp/includes/Rest_Controller.php

```diff
@@ -0,0 +1,354 @@
+			"AN" => "Netherlands Antilles",
+			"NC" => "New Caledonia",
+			"NZ" => "New Zealand",
+			"NI" => "Nicaragua",
+			"NE" => "Niger",
+			"NG" => "Nigeria",
+			"NU" => "Niue",
+			"NF" => "Norfolk Island",
+			"MP" => "Northern Mariana Islands",
+			"NO" => "Norway",
+			"OM" => "Oman",
+			"PK" => "Pakistan",
+			"PW" => "Palau",
+			"PS" => "Palestinian Territory, Occupied",
+			"PA" => "Panama",
+			"PG" => "Papua New Guinea",
+			"PY" => "Paraguay",
+			"PE" => "Peru",
+			"PH" => "Philippines",
+			"PN" => "Pitcairn",
+			"PL" => "Poland",
+			"PT" => "Portugal",
+			"PR" => "Puerto Rico",
+			"QA" => "Qatar",
+			"RE" => "Reunion",
+			"RO" => "Romania",
+			"RU" => "Russian Federation",
+			"RW" => "Rwanda",
+			"SH" => "Saint Helena",
+			"KN" => "Saint Kitts and Nevis",
+			"LC" => "Saint Lucia",
+			"PM" => "Saint Pierre and Miquelon",
+			"VC" => "Saint Vincent and the Grenadines",
+			"WS" => "Samoa",
+			"SM" => "San Marino",
+			"ST" => "Sao Tome and Principe",
+			"SA" => "Saudi Arabia",
+			"SN" => "Senegal",
+			"CS" => "Serbia and Montenegro",
+			"SC" => "Seychelles",
+			"SL" => "Sierra Leone",
+			"SG" => "Singapore",
+			"SK" => "Slovakia",
+			"SI" => "Slovenia",
+			"SB" => "Solomon Islands",
+			"SO" => "Somalia",
+			"ZA" => "South Africa",
+			"GS" => "South Georgia and the South Sandwich Islands",
+			"ES" => "Spain",
+			"LK" => "Sri Lanka",
+			"SD" => "Sudan",
+			"SR" => "Suriname",
+			"SJ" => "Svalbard and Jan Mayen",
+			"SZ" => "Swaziland",
+			"SE" => "Sweden",
+			"CH" => "Switzerland",
+			"SY" => "Syrian Arab Republic",
+			"TW" => "Taiwan, Province of China",
+			"TJ" => "Tajikistan",
+			"TZ" => "Tanzania, United Republic of",
+			"TH" => "Thailand",
+			"TL" => "Timor-Leste",
+			"TG" => "Togo",
+			"TK" => "Tokelau",
+			"TO" => "Tonga",
+			"TT" => "Trinidad and Tobago",
+			"TN" => "Tunisia",
+			"TR" => "Turkey",
+			"TM" => "Turkmenistan",
+			"TC" => "Turks and Caicos Islands",
+			"TV" => "Tuvalu",
+			"UG" => "Uganda",
+			"UA" => "Ukraine",
+			"AE" => "United Arab Emirates",
+			"GB" => "United Kingdom",
+			"US" => "United States",
+			"UM" => "United States Minor Outlying Islands",
+			"UY" => "Uruguay",
+			"UZ" => "Uzbekistan",
+			"VU" => "Vanuatu",
+			"VE" => "Venezuela",
+			"VN" => "Viet Nam",
+			"VG" => "Virgin Islands, British",
+			"VI" => "Virgin Islands, U.s.",
+			"WF" => "Wallis and Futuna",
+			"EH" => "Western Sahara",
+			"YE" => "Yemen",
+			"ZM" => "Zambia",
+			"ZW" => "Zimbabwe"
+		);
+		
+		$data = array();
+		foreach( $countries as $country_code => $country_name ) {
+			$data[] = array(
+				'code' => $country_code,
+				'name' => $country_name
+			);
+		}
+
+		return rest_ensure_response( $data );
+	}
+
+	public function existing_data() {
+		$configuration_service = Wordlift_Configuration_Service::get_instance();
+		$publisher_id = $configuration_service->get_publisher_id();
+
+		if ( empty($publisher_id) || $publisher_id == "(none)" ) {
```

Comment:
`===` use strict comparison. The side effects may be devastating.

Reply:
Thanks. I've changed it.